ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -105,27 +106,52 @@
                                    [(format "A find value must be supplied when the update is of type %s"
                                             update-type)]))))
 
-(defn- get-provider-concept-ids
-  "Returns a list of collection concept ids for a given provider-id"
+(defn- get-provider-collection-concept-ids
+  "Returns a list of collection concept ids for a given provider-id.
+   Throws exception if non are returned."
   [context provider-id]
   (let [concepts (mdb/find-concepts context 
-                                    {:provider-id provider-id} 
-                                    :collection)]
-    (when (seq concepts)
-      (distinct (map :concept-id concepts)))))
+                                    {:provider-id provider-id
+                                     :latest "true"} 
+                                    :collection)
+        concepts (remove :deleted concepts)]
+    (if (seq concepts)
+      (distinct (map :concept-id concepts))
+      (errors/throw-service-errors
+        :bad-request
+        ["No concept-ids are associated with the provider-id."]))))
+
+(defn- get-collection-concept-id-validation-err-msgs
+  "Returns the concept-id validation msgs"
+  [concept-ids]
+  (for [id concept-ids
+        :let [err-msg (concepts/concept-id-validation id)
+              err-msg 
+               (if err-msg
+                 err-msg
+                 (when-not (string/starts-with? id "C")
+                   [(format "Collection concept-id [%s] is invalid, must start with C" id)]))] 
+        :when err-msg]
+    err-msg))
+
+(defn- validate-concept-ids
+  "Throws exception if there exists invalid
```

Comment:
nit: put the docstring in one line.

Reply:
Changed to "Raise error if there exists invalid concept-ids."